cdap-docs/developers-manual/source/building-blocks/mapreduce-programs.rst

```diff
@@ -136,7 +136,8 @@ MapReduce and Datasets
 .. rubric: Reading and Writing to Datasets from a MapReduce program
 
 Both a CDAP ``mapper`` and ``reducer`` can directly read or write to a dataset, using
-one of these options:
+one of the following options. Note that options 2 and 3 can also be used for a Partitioner
+and Comparator class, if configured on the MapReduce job:
```

Comment:
to be consistent with the first sentence, should it be ``partitioner`` and ``comparator`` (without class)?